ncm-postgresql/src/main/pan/components/postgresql/schema.pan

```diff
@@ -86,7 +86,8 @@ type postgresql_mainconfig = {
     "bytea_output" ? string     # "hex"
     "check_function_bodies" ? boolean    # true
     "checkpoint_completion_target" ? string     # 0.5
-    "checkpoint_segments" ? long       # 3
+    "checkpoint_segments" ? long with {deprecated(
```

Comment:
@jrha @ned21 is there a more legant way to split up lines in pan?
wpouter, have you tried with a backslash + nelwine linebreak, eg
```pan
checkpoint_segments" ? long with \
    deprecated(0, 'checkpoint_segments is deprecated since Postgres version 9.5.'); true; } # 3
```
(no space after the `\`)